lib/index.js

```diff
@@ -162,3 +191,15 @@ Ramen.prototype.group = function(group) {
 
   window.Ramen.go();
 };
+
+Ramen.prototype.page = function() {
+  window.Ramen.go();
+};
+
+Ramen.prototype.track = function(track) {
+  window.Ramen.Api.track_named(track.event());
+};
+
+Ramen.version = function() {
```

Comment:
@ndhoule yes sir. I sent this in a note to @amillet89 through partners@segment.com: we want to support the `page` call, but right now RamenJS automatically supports single page apps by simply checking the `window.location.href` for changes every 100ms. Hacky, but it works.

Once the Segment Ramen integration starts to support `page`, we obviously want to turn off this interval check. I would check `analytics.Integrations.Ramen.version()` and turn off that href observer.